### PR TITLE
Style fixes and general code patchups

### PR DESCRIPTION
**Pull request by kaylynn234**
This PR:
-Makes code style consistent and more in line with PEP 8.
-Moves a few functions around to places where they'll work as intended.
-Generally cleans things up.

### CODE REVIEW

File: cogs/moderation.py [hunk 1]
```diff
@@ -6,151 +6,155 @@
...
+        await ctx.send("This guild" + (f"'s join message is: \n{joinmsg}" if joinmsg else " has no join message."))
 
-    @joinmsg.command()
+    @joinmsg.command(name="set")
     @commands.has_permissions(manage_guild=True)
-    async def set(self, ctx, *, joinmessage:str):
-        ''' Set the guild's custom join message. '''
-        if (len(joinmessage) > 1350):
-            return await ctx.send('Guild join messages may only be a max of 1,350 characters long.')
+    async def set_join_message(self, ctx, *, joinmessage: str):
```
Comment (keli5) [inline cogs/moderation.py:74]: why change this from `async def set(...)`?

Comment (kaylynn234) [reply cogs/moderation.py:74]: Naming it "set" shadows the builtin datatype _set_ . Same story with the command previously named _list_. Both are generally bad practice, and they also mess up the syntax highlighting in VSC (which is the real evil)

Comment (keli5) [reply cogs/moderation.py:74]: Oh, got it.
